conf/config-default.yaml

```diff
@@ -438,6 +438,9 @@ plugin_attr:
     export_addr:
       ip: 127.0.0.1
       port: 9091
+        #    labels:
+        #remote_addr: $remote_addr
+        #request_method: $request_method
```

Comment:
This function is userful for us. we add metrics too on our code as hard code way.
Can you list ot give a link which labels are support ?